cli/npm/byonm.rs

```diff
@@ -0,0 +1,271 @@
+use std::borrow::Cow;
```

Comment:
Missing copyright - surprising that copyright checker didn't catch it

Reply:
Seems it was broken by https://github.com/denoland/deno/pull/20682